src/map/zone_entities.cpp

```diff
@@ -163,7 +163,7 @@ void CZoneEntities::TryAddToNearbySpawnLists(CBaseEntity* PEntity)
                 case TYPE_PC:
                 {
                     PCurrentChar->SpawnPCList[PEntity->id] = PEntity;
-                    PCurrentChar->updateEntityPacket(PEntity, ENTITY_SPAWN, UPDATE_ALL_CHAR);
+                    PCurrentChar->updateCharPacket(static_cast<CCharEntity*>(PEntity), ENTITY_SPAWN, UPDATE_ALL_CHAR);
```

Comment:
This must have been a needle in a haystack. good job

Reply:
TY, before this issue was reported I was actually working on combining `updateCharPacket` into `updateEntityPacket` so it was pretty fresh in my mind. The naming is pretty confusing so it'll be nice to just do it like it was here and have everything use `updateEntityPacket`. But the branch I was working on didn't have that merged so I was dealing with different layers of bugs interacting without realizing it for a while.

Reply:
PEntity and PCurrentChar are the same pointer here, static_cast is just type system shenanigans. This should just be PCurrentChar?

Reply:
Oh shit, I see it! YOU ABSOLUTE DETECTIVE!